src/proto_bcd/findings/finding.py

```diff
@@ -73,3 +74,13 @@ def to_dict(self):
     def get_message(self):
         format_parameters = self.to_dict()
         return templates[self.category].format(**format_parameters)
+
+    def get_suggested_commit(self):
+        format_parameters = self.to_dict()
+        # add relative context name to make conventional commits less verbose
+        # when the changes all have the same package name
+        format_parameters["rel_context"] = next(
+            (self.context[i:] for i, c in enumerate(self.context) if c.isupper()),
+            self.context,
+        )
+        return suggested_commit_templates[self.category].format(**format_parameters)
```

Comment:
Would it be possible to to default to the the `Finding.message` if the category isn't configured with a template? Is that a reasonable fall back?